include/dali/core/common.h

```diff
@@ -49,11 +49,12 @@ using std::vector;
 // Common types
 using uint8 = uint8_t;
 using uint16 = uint16_t;
-using int16 = int16_t;
-using int64 = int64_t;
+using uint32 = uint32_t;
 using uint64 = uint64_t;
+using int8 = int8_t;
+using int16 = int16_t;
 using int32 = int32_t;
-using uint32 = uint32_t;
+using int64 = int64_t;
```

Comment:
Can we remove those instead? They're not used that much and many places in the code use the standard names anyway.

Reply:
Removed all the types and converted all the occurrence to `*_t`.